packages/elements-angular/elements/README.md

```diff
@@ -12,25 +12,35 @@ Please refer to the top level [README](../../../README.md) to see the available
 
 ### Adding or Removing Components
 
-Everytime a new component ist added or removed from the `@inovex.de/elements` package you must ensure that
-the following files are written after running the `build` command:
+Everytime a new component ist added or removed from the `@inovex.de/elements` package you must ensure that the following files are written after running the `build` command:
 
 - `src/directives/proxies-list.ts`
 - `src/directives/proxies.ts`
 
-These files are updated automatically. This ensures that the component is available for the consumer in
-the Angular world. If the component needs a custom value accessor to support the `ngModel` directive,
-please update the following files also:
+These files are updated automatically. 
+This ensures that the component is available for the consumer in the Angular world. 
+If the component needs a custom value accessor to support the `ngModel` directive, please update the following files also:
 
 - `src/public-api` (modify)
 - `src/ino-elements.module.ts` (modify)
 - `src/directives/control-value-accesors/ino-COMPONENT-NAME-value-accessor.directive.ts` (create)
 
+### Schematics
+For a detailed explanation of schematics in Angular, see [this guide](https://angular.io/guide/schematics).
+
+At this moment, `ng-add` is the only schematic this library provides.
+All the related files are located under `/schematics`.
+
+In order to test the functionality of the schematics, we make use of our linking tool `yalc`:
+1. Add a new schematic or make changes to the existing one
+2. Run `publish:yalc` at the root of this project and select angular
+3. In your consuming project, install [yalc](https://github.com/wclr/yalc) and run `yalc link @inovex.de/elements-angular`
+4. Run `ng add @inovex.de/elements-angular`
```

Comment:
Does not work as described, even on starting from clean state.

<img width="350" alt="grafik" src="https://user-images.githubusercontent.com/12165722/121912002-7f222300-cd30-11eb-9951-da9f2b51f6b5.png">

Reply:
I think I found the error cause: I did not check if the object exists in your `angular.json` before adding the icons.

Reply:
Tested. The error is fixed an the angular.json is edited appropriate.